plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetadataFileOperations.java

```diff
@@ -496,6 +496,24 @@ public void testRemoveOrphanFiles()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testInformationSchemaColumns()
+    {
+        int tables = 3;
+        for (int i = 0; i < tables; i++) {
+            assertUpdate("CREATE TABLE test_select_i_s_columns" + i + "(id VARCHAR, age INT)");
+        }
+
+        assertFileSystemAccesses("SELECT * FROM information_schema.columns WHERE table_name LIKE 'test_select_i_s_columns%'",
+                ImmutableMultiset.<FileOperation>builder()
+                        .addCopies(new FileOperation(METADATA_JSON, INPUT_FILE_NEW_STREAM), 3)
```

Comment:
Would be nice to show that if you have a table that does not match the filter we don't grab the metadata json file. Similarly a predicate on table_schema should filter the same way.